src/modules/sub-items/sub.items.module.js

```diff
@@ -455,28 +455,42 @@ export default class SubItemsModule extends Component {
                 'sub_items'
             );
             const notificationMessage = Translator.trans(
-                /*@Desc("%notMovedCount% of the %totalCount% selected item(s) could not be moved because you do not have proper user permissions. <u><a class='ez-notification-btn ez-notification-btn--show-modal'>Click here for more information.</a></u><br>Please contact your Administrator to obtain permissions.")*/ 'bulk_move.error.message',
+                /*@Desc("%notMovedCount% of the %totalCount% selected item(s) could not be moved because you do not have proper user permissions. {{ placeholder }} Please contact your Administrator to obtain permissions.")*/ 'bulk_move.error.message',
                 {
                     notMovedCount: notMovedLocations.length,
                     totalCount: movedLocations.length + notMovedLocations.length,
                 },
                 'sub_items'
             );
+            const rawPlaceholdersMap = {
+                placeholder: Translator.trans(
```

Comment:
I would use a more explanatory key name. What if we will have to add the second placeholder?